hardware/ThermostatHardware.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <iostream>
```

Comment:
I don't see 'iostream' objects in this header files. But I'm missing string, vector, algorithm (std::find) includes

Reply:
Ok!done